src/main/charts/jira/templates/_helpers.tpl

```diff
@@ -93,6 +93,19 @@ The command that should be run by the nfs-fixer init container to correct the pe
 {{- end }}
 {{- end }}
 
+{{ define "jira.volumeMounts" }}
+- name: local-home
+  mountPath: {{ .Values.volumes.localHome.mountPath | quote }}
+- name: local-home
```

Comment:
I wonder if we really need two mounts. I know it's cleaner from the perspective of the sidecar, but does that 'perspective' matter? It does make the charts more complicated.

Reply:
The alternative is to have an extra volume just to mount the tomcat logs, and for consistency that extra volume would need to be a PV. That's a lot of extra baggage when we can just double-mount the same volume instead.